### PR TITLE
feat: declarativly toggle visibility

### PR DESCRIPTION
**Pull request by jon-coffey**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

visibility (by triggering show/hide) can now be done by declarative angular binding ([isVisible]="true|false")

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Now show() and hide() have to be called manually.

Issue Number: https://github.com/ngneat/helipopper/discussions/60

## What is the new behavior?

Now [isVisible]="true|false" can be used to call show()/hide()

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information

I tried limit the change to a minimum. Please suggest if something could be done better.

### CODE REVIEW

**Comment by jon-coffey** [comment]
Havn't found the issue with the failing pipeline. Any suggestions? It has something to do with the routing. It tells that /is-visible is not found.

Running npm run e2e locally works fine.
Which command is run in the CI?

**Comment by jon-coffey** [comment]
Ok. A little bit further I see that running serve:playground:static doesn't support the route because of serving without webpack-development server. Any idea how to get the route thing running? (Or replace the second route?)

**Comment by jon-coffey** [comment]
Hi
1. I really tried the rebase but now it's the same message twice. Bad? Any way to do it better?
2. I mitigated failing test by now using angular-http-server instead of the serve package. that allows fallback to index.html in spa

**Comment by jon-coffey** [comment]
Finally. CI is green. Squashed commits look fine. Hope now it's ok. Just tell me if there is more to it.

File: cypress/integration/helipopper.spec.ts [hunk 1]
```diff
@@ -105,4 +105,17 @@ describe('@ngneat/helipopper', () => {
         .should('exist');
     });
   });
+
+  
+  describe('isVisible', () => {
```
Comment (NetanelBasal) [inline cypress/integration/helipopper.spec.ts:110]: Please move it to the is visible spec file, and change the file/dir name to dash case

Comment (jon-coffey) [reply cypress/integration/helipopper.spec.ts]: Done

**Pull request merged**